File: app/src/main/java/com/luck/pictureselector/SimpleActivity.java
package com.luck.pictureselector;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import androidx.appcompat.app.AppCompatActivity;

public class SimpleActivity extends AppCompatActivity implements View.OnClickListener {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_other);
        Button btn_activity = findViewById(R.id.btn_activity);
        Button btn_inject_fragment = findViewById(R.id.btn_inject_fragment);
        Button btn_only_query_data = findViewById(R.id.btn_only_query_data);
        btn_activity.setOnClickListener(this);
        btn_inject_fragment.setOnClickListener(this);
        btn_only_query_data.setOnClickListener(this);
    }

    @Override
    public void onClick(View v) {
        if (v.getId() == R.id.btn_activity) {
            startActivity(new Intent(SimpleActivity.this, MainActivity.class));
        } else if (v.getId() == R.id.btn_inject_fragment){
            startActivity(new Intent(SimpleActivity.this, InjectFragmentActivity.class));
        } else if (v.getId() == R.id.btn_only_query_data){
            startActivity(new Intent(SimpleActivity.this, OnlyQueryDataActivity.class));
        }
    }
}

File: app/src/main/java/com/luck/pictureselector/SimpleActivity.kt
package com.luck.pictureselector

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity

class SimpleActivity : AppCompatActivity(), View.OnClickListener {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_simple)
        findViewById<Button>(R.id.btn_activity).setOnClickListener(this)
        findViewById<Button>(R.id.btn_inject_fragment).setOnClickListener(this)
        findViewById<Button>(R.id.btn_only_query_data).setOnClickListener(this)
    }

    override fun onClick(v: View) {
        when (v.id) {
            R.id.btn_activity -> {
                startActivity(Intent(this@SimpleActivity, MainActivity::class.java))
            }
            R.id.btn_inject_fragment -> {
                startActivity(Intent(this@SimpleActivity, InjectFragmentActivity::class.java))
            }
            R.id.btn_only_query_data -> {
                startActivity(Intent(this@SimpleActivity, QueryDataActivity::class.java))
            }
        }
    }
}